Foundation/NSGeometry.swift

```diff
@@ -25,6 +25,20 @@ public struct CGPoint {
     }
 }
 
+extension CGPoint {
```

Comment:
Why is this an extension? Is this a convention I'm not aware of?  I would have added to the `struct CGPoint {}` definition just above.